service/src/test/java/org/sunbird/learner/actors/UserManagementActorTestExtended.java

```diff
@@ -0,0 +1,398 @@
+import org.sunbird.actor.router.RequestRouter;
+import org.sunbird.actor.service.SunbirdMWService;
+import org.sunbird.actorutil.InterServiceCommunication;
+import org.sunbird.actorutil.InterServiceCommunicationFactory;
+import org.sunbird.actorutil.impl.InterServiceCommunicationImpl;
+import org.sunbird.actorutil.systemsettings.impl.SystemSettingClientImpl;
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.cassandraimpl.CassandraOperationImpl;
+import org.sunbird.common.ElasticSearchUtil;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.ActorOperations;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.ProjectUtil;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.request.UserRequestValidator;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.content.util.ContentStoreUtil;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.learner.util.DataCacheHandler;
+import org.sunbird.learner.util.Util;
+import org.sunbird.services.sso.SSOManager;
+import org.sunbird.services.sso.SSOServiceFactory;
+import org.sunbird.services.sso.impl.KeyCloakServiceImpl;
+import org.sunbird.telemetry.util.TelemetryUtil;
+import org.sunbird.user.actors.UserManagementActor;
+import org.sunbird.user.dao.UserExternalIdentityDao;
+import org.sunbird.user.dao.impl.UserExternalIdentityDaoImpl;
+import org.sunbird.user.service.UserService;
+import org.sunbird.user.service.impl.UserServiceImpl;
+import org.sunbird.user.util.UserUtil;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+  ServiceFactory.class,
+  Util.class,
+  ElasticSearchUtil.class,
+  SunbirdMWService.class,
+  UserRequestValidator.class,
+  UserUtil.class,
+  DataCacheHandler.class,
+  CassandraOperation.class,
+  InterServiceCommunication.class,
+  TelemetryUtil.class,
+  InterServiceCommunicationFactory.class,
+  InterServiceCommunicationImpl.class,
+  SystemSettingClientImpl.class,
+  RequestRouter.class,
+  UserServiceImpl.class,
+  UserService.class,
+  UserExternalIdentityDao.class,
+  UserExternalIdentityDaoImpl.class,
+  SSOServiceFactory.class,
+  SSOManager.class,
+  KeyCloakServiceImpl.class,
+  ContentStoreUtil.class
+})
+@PowerMockIgnore({"javax.management.*", "javax.crypto.*", "javax.net.ssl.*", "javax.security.*"})
+public class UserManagementActorTestExtended {
+  private static ActorSystem system;
+  private static final Props props = Props.create(UserManagementActor.class);
+  private static String userId = "testUserId";
+  private static CassandraOperation cassandraOperation;
+  private static UserService userService;
+  private static InterServiceCommunication interServiceCommunication;
+  private static SystemSettingClientImpl systemSettingClient;
+  private static UserExternalIdentityDaoImpl userExtDao;
+  private static SSOManager ssoManager;
```

Comment:
variables which are used just to add one mock rule etc, or are used in one function call (not across multiple functions) can be declared locally to that function